package.json

```diff
@@ -79,6 +80,8 @@
     "karma-webpack": "1.7.0",
     "lodash": "4.3.0",
     "mocha": "2.4.5",
+    "preact": "^8.1.0",
```

Comment:
Used in `.storybook/webpack.config.js` for the `storybook:preact` script in `package.json`.